homeassistant/components/sensor/plex.py

```diff
@@ -116,8 +116,25 @@ def update(self):
         now_playing = []
         for sess in sessions:
             user = sess.usernames[0] if sess.usernames is not None else ""
-            title = sess.title if sess.title is not None else ""
-            year = sess.year if sess.year is not None else ""
-            now_playing.append((user, "{0} ({1})".format(title, year)))
+            if sess.TYPE == 'movie':
```

Comment:
What happens if neither a movie nor an episode is played? I use Plex as the source for music. Haven't looked deeper into how Plex works, but I think there should be an `else` for everything besides movies and episodes.

Reply:
Oh you are totally right. I will work on it the next days

Reply:
good catch.. and this is why i should not review code straight after work :)